pkg/noderesourcetopology/TESTS.md

```diff
@@ -0,0 +1,78 @@
+| 12        | 1               | R < NUMA                         | N > 1      | NUMA < SUM(R) < Node && R < NUMA | Running |
+| 13        | 1               | R < NUMA                         | N > 1      | SUM(R) > Node && R < NUMA        | Pending |
+| 14        | 1               | R < NUMA                         | N > 1      | NUMA < R < Node                  | Pending |
+| 15        | N > 1           | R < NUMA                         | 1          | R < NUMA                         | Running |
+| 16        | N > 1           | R < NUMA                         | 1          | NUMA < R < Node                  | Pending |
+| 17        | N > 1           | R < NUMA                         | N > 1      | R < NUMA                         | Running |
+| 18        | N > 1           | R < NUMA                         | N > 1      | NUMA < SUM(R) < Node && R < NUMA | Running |
+| 19        | N > 1           | R < NUMA                         | N > 1      | SUM(R) > Node && R < NUMA        | Pending |
+| 20        | N > 1           | R < NUMA                         | N > 1      | NUMA < R < Node                  | Pending |
+| 21        | N > 1           | NUMA < SUM(R) < Node && R < NUMA | 1          | R < NUMA                         | Running |
+| 22        | N > 1           | NUMA < SUM(R) < Node && R < NUMA | 1          | NUMA < R < Node                  | Pending |
+| 23        | N > 1           | NUMA < SUM(R) < Node && R < NUMA | N > 1      | R < NUMA                         | Pending |
+| 24        | N > 1           | NUMA < SUM(R) < Node && R < NUMA | N > 1      | NUMA < SUM(R) < Node && R < NUMA | Running |
+| 25        | N > 1           | NUMA < SUM(R) < Node && R < NUMA | N > 1      | SUM(R) > Node && R < NUMA        | Pending |
+| 26        | N > 1           | NUMA < SUM(R) < Node && R < NUMA | N > 1      | NUMA < R < Node                  | Pending |
+| 27        | N > 1           | SUM(R) > Node && R < NUMA        | 1          | R < NUMA                         | Running |
+| 28        | N > 1           | SUM(R) > Node && R < NUMA        | 1          | NUMA < R < Node                  | Pending |
+| 29        | N > 1           | SUM(R) > Node && R < NUMA        | N > 1      | R < NUMA                         | Running |
+| 30        | N > 1           | SUM(R) > Node && R < NUMA        | N > 1      | NUMA < SUM(R) < Node && R < NUMA | Running |
+| 32        | N > 1           | SUM(R) > Node && R < NUMA        | N > 1      | SUM(R) > Node && R < NUMA        | Pending |
+| 33        | N > 1           | SUM(R) > Node && R < NUMA        | N > 1      | NUMA < R < Node                  | Pending |
+| 34        | N > 1           | NUMA < R < Node                  | NVM        | NVM                              | Pending |
+
+### Test Tiers
+
+#### Tier1
+Contains the most fundamental and critical tests.
+Init containers with multi/single-containers placement against multiple NUMAs.
```

Comment:
Init and app containers or only init containers? I guess the former, right?

Reply:
yes I'll mention that explicitly